packages/expo-module-scripts/package.json

```diff
@@ -25,7 +25,6 @@
     "@babel/cli": "^7.1.2",
     "@expo/npm-proofread": "^1.0.1",
     "@types/jest": "^23.3.9",
-    "babel-preset-expo": "^5.0.0",
```

Comment:
If we remove `babel-preset-expo` from `expo-module-scripts/package.json`, shouldn't we also remove it from https://github.com/expo/expo/blob/master/packages/expo-module-scripts/babel.config.base.js#L4?

Reply:
Oops, I didn't mean to remove this here. Thanks for catching this.